src/js/structs/Item.js

```diff
@@ -7,6 +7,9 @@ module.exports = class Item {
     this._itemData = item;
   }
 
+  toJSON() {
+    return JSON.stringify(this._itemData);
```

Comment:
Let's do `this.get()` to not access the internal property directly and use the API in stead.